scripts/hls.py

```
@@ -0,0 +1,312 @@
+            "-i",
+            url,
+            "-c",
+            "copy",
+            "-bsf:a",
+            "aac_adtstoasc",
+            output_file,
+        ],
+    ]
+    for cmd in commands:
+        try:
+            result = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)